src/parser.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import annotations
+import requests as rq
+import json
+from bs4 import BeautifulSoup as bs
+import configuration as cf
+from transformers import pipeline
+
+models: dict = {}
+
+
+def load_models():
+    question_name = "deepset/roberta-base-squad2"
+    models["qa"] = pipeline("question-answering", model=question_name, tokenizer=question_name)
+    translation_name = "Helsinki-NLP/opus-mt-ru-en"
+    models["trans"] = pipeline("translation", model=translation_name, tokenizer=translation_name)
+    classifier_name = "joeddav/xlm-roberta-large-xnli"
+    models["class"] = pipeline("zero-shot-classification", model=classifier_name)
```

Comment:
Well, as an ad-hoc solution this is probably okay. I would add a check that the dict is empty before loading, to handle cases with multiple function calls.

But a more accurate solution is to use classes, you can wrap your parser into the class that initializes models and provide an open interface for parsing. This also would help with optimizing some other functions in this file.

Reply:
I completely agree. Wrapped up in a separate class. Looks better